src/components/UsernameEntry/UsernameEntry.scss

```diff
@@ -25,3 +25,7 @@ input {
   width: 100%;
   height: 232px;
 }
+
+.username-icon {
```

Comment:
icon styles are already defined in list selection (separately for type of icon, might be worth moving this to that to make it organised)